modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -559,6 +563,17 @@ public boolean stopRaftNodes(ReplicationGroupId groupId) {
         });
     }
 
+    @Override
+    public void destroyRaftNodeStorages(RaftNodeId nodeId, RaftGroupOptions groupOptions) {
```

Comment:
An idea came up at the very end of the review. Let's say for some error or reason we didn't delete the directory or something else, we don't throw an exception for errors and only write a message to the log and that's it. Maybe we should somehow let the user fix it later via the console or should we automatically try again, what do you think? It's not necessary to do it now, maybe later.

Reply:
I added a TODO to address this issue separately